src/blog/templates/blog/includes/comments.html

```diff
@@ -181,6 +198,60 @@ <h2 class="text-2xl">Comments</h2>
         });
         li.appendChild(likeButton);
 
+        likeButton.addEventListener("mouseover", function () {
+            const commentId = comment.id
+            fetch(`/blog/api/comment-likes/${commentId}/`, {
+                method: "GET",
+                headers: {
+                    "Content-Type": "application/json",
+                    "X-CSRFToken": document.querySelector(
+                        'input[name="csrfmiddlewaretoken"]',
+                    ).value,
+                },
+            })
```

Comment:
This seems like a pretty expensive operation. What if you shake your mouse a bit, will it fetch the data 5 times in one second? Thinking out loud: How actual should the comment like information be? Can't you re-fetch the information whenever you like or dislike a comment?